integration_tests/mockApis/licence.ts

```diff
@@ -1168,4 +1168,18 @@ export default {
       },
     })
   },
+
+  stubGetBankHolidays: (): SuperAgentRequest => {
```

Comment:
ah.. not actually passing dates through to jsonBody here

Reply:
Yup, spotted that yesterday. Done.